Internal name of union variables

In according to gometalinter:
```
+main.go:441:1: receiver name should be a reflection of its identity; don't use generic names such as
 "this" or "self"
+main.go:444:1: receiver name should be a reflection of its identity; don't use generic names such as
 "this" or "self"
+main.go:448:1: receiver name should be a reflection of its identity; don't use generic names such as
 "this" or "self"
+main.go:452:1: receiver name should be a reflection of its identity; don't use generic names such as
 "this" or "self"
```
Need modify internal names in union

--- a/transpiler/union.go
+++ b/transpiler/union.go
@@ -39,31 +39,31 @@ type {{ .Name }} struct{
 	arr      [{{ .Size }}]byte
 }
 
-func (self *{{ .Name }}) cast(t reflect.Type) reflect.Value {
-	return reflect.NewAt(t, unsafe.Pointer(&self.arr[0]))
+func (unionVar*{{ .Name }}) cast(t reflect.Type) reflect.Value {
+	return reflect.NewAt(t, unsafe.Pointer(&unionVar.arr[0]))
 }
 
-func (self *{{ .Name }}) assign(v interface{}){
+func (unionVar*{{ .Name }}) assign(v interface{}){
 	value := reflect.ValueOf(v).Elem()
-	value.Set(self.cast(value.Type()).Elem())
+	value.Set(unionVar.cast(value.Type()).Elem())
 }
 
-func (self *{{ .Name }}) UntypedSet(v interface{}){
+func (unionVar*{{ .Name }}) UntypedSet(v interface{}){
 	value := reflect.ValueOf(v)
-	self.cast(value.Type()).Elem().Set(value)
+	unionVar.cast(value.Type()).Elem().Set(value)
 }
 
 {{ range .Fields }}
 // Get{{ .Name }} - return value of {{ .Name }}
-func (self *{{ $.Name }}) Get{{ .Name }} () (res {{ .TypeField }}){
-	self.assign(&res)
+func (unionVar*{{ $.Name }}) Get{{ .Name }} () (res {{ .TypeField }}){
+	unionVar.assign(&res)
 	return
 }
 
 // Set{{ .Name }} - set value of {{ .Name }}
-func (self *{{ $.Name }}) Set{{ .Name }} (v {{ .TypeField }}) {{ .TypeField }}{
-	self.value = v // added for avoid GC removing pointers in union
-	self.UntypedSet(v)
+func (unionVar*{{ $.Name }}) Set{{ .Name }} (v {{ .TypeField }}) {{ .TypeField }}{
+	unionVar.value = v // added for avoid GC removing pointers in union
+	unionVar.UntypedSet(v)
 	return v
 }
 {{ end }}

